app/views/course/question_assessments/_question_assessment.html.slim

```diff
@@ -17,4 +17,6 @@
             = edit_button(edit_path)
             = delete_button(delete_path)
         h4 = format_inline_text(question_assessment.display_title)
-        = format_html(question.description)
+        span.label.label-primary
+          = format_inline_text(question_assessment.question.actable.type)
```

Comment:
Couple of points: 
 1. Shall we just standardise the method name to `#question_type` for all types?
 2. We can make this shorter to `question_asessment.question.question_type`, by adding this  `delegate :question_type, to: :actable`

After this it looks good 👍